frontend/app/components/inplace-edit/directives/edit-pane/edit-pane.directive.js

```diff
@@ -251,14 +165,30 @@ function InplaceEditorEditPaneController($scope, $element, $location, $timeout,
   });
 
   $scope.$on('form.updateRequired', function() {
-    var submit = $q.defer();
-    vm.updateWorkPackageForm(submit);
+    inplaceEditStorage.updateWorkPackageForm();
   });
 
   $scope.$on('workPackageRefreshed', function() {
     vm.discardEditing();
-    EditableFieldsState.isBusy = false;
+  });
+
+  $scope.$on('inplaceEditMultiStorage.save.workPackage', function (event, promise) {
+    promise.then(function() {
+      $location.hash(null);
```

Comment:
Right, it did not appear to me, that those things would only make sense, if you actually click or somehow make the field active.